src/main/java/com/exasol/adapter/document/querypredicate/QueryPredicateToBooleanExpressionConverter.java

```diff
@@ -51,10 +52,33 @@ public void visit(final NotPredicate notPredicate) {
         public void visit(final ColumnLiteralComparisonPredicate columnLiteralComparisonPredicate) {
             final ValueExpression literal = SqlLiteralToValueExpressionConverter.getInstance()
                     .convert(columnLiteralComparisonPredicate.getLiteral());
-            this.result = new Comparison(ComparisonOperator.EQUAL,
+            this.result = new Comparison(convertOperator(columnLiteralComparisonPredicate.getOperator()),
                     column(columnLiteralComparisonPredicate.getColumn().getExasolColumnName()), literal);
         }
 
+        private ComparisonOperator convertOperator(final Operator operator) {
+            switch (operator) {
+            case GREATER:
+                return ComparisonOperator.GREATER_THAN;
+            case GREATER_EQUAL:
+                return ComparisonOperator.GREATER_THAN_OR_EQUAL;
+            case EQUAL:
+                return ComparisonOperator.EQUAL;
+            case LESS_EQUAL:
+                return ComparisonOperator.LESS_THAN_OR_EQUAL;
+            case LESS:
+                return ComparisonOperator.LESS_THAN;
+            case NOT_EQUAL:
+                return ComparisonOperator.NOT_EQUAL;
+            case LIKE:
+                throw new UnsupportedOperationException(
+                        "LIKE is not yet supported. See https://github.com/exasol/sql-statement-builder/issues/91");// TODO
```

Comment:
Skipping this, I assume you know that the issue is fixed.

Reply:
yes. I will fix in an upcoming PR.